concepts/solvers.mdx

```diff
@@ -4,22 +4,27 @@ sidebarTitle: "Solvers"
 ---
 
 
+
 # Preconfirmations through Solvers
 
-Entities that specialize in solving specific problems are often denoted as "Solvers". In this context, we define a solver to be an entity that utilizes mev-commit to provide preconfirmations to regular users. Solvers aim to abstract the details mentioned above from the end-users, who may lack the expertise to make these decisions on their own. The rest of this document discusses how solvers can help end-users secure pre-confirmations for their transaction bundles.
+This article explains how third parties, referred to as solvers, can use mev-commit as a tool to offer preconfirmation services to users. By issuing preconfirmations, solvers can fulfill a range of user requests, from basic intents such as fast bundle inclusion in the chain to complex operations involving multiple chains. Solvers aim to abstract these detailed processes from end-users, who may lack the expertise to perform these operations independently. Here we discuss how solvers can help users secure preconfirmations for their transaction bundles.
+
+Preconfirmations serve various purposes. They can benefit a searcher looking to profit from an arbitrage opportunity or a regular user wanting to sell assets on Ethereum while buying assets on Solana at the same time. This can be a daunting task for users who lack the expertise to interact directly with mev-commit to request and acquire the necessary preconfirmations from providers.
+
+<Note>We assume there is a single solver (called the "leader") performing operations on behalf of the users. The leader can be selected in various ways, such as through an auction or a lottery.</Note>
 
-## Issuing and Sourcing Preconfs
+The reason for considering a single solver is that if a user requests the same intent from multiple solvers, it requires coordination between them to fulfill the intent profitably. For example, if two solvers are trying to fulfil an intent that involves transactions on two separate chains, one solver might manage to source preconfirmations from both chains while the other solver only sources one. In this case, both solvers will have to pay for the preconfirmations to the providers, but only one solver will actually fulfil the user's intent and get paid by the user. Coordination between solvers is out of scope of the mev-commit protocol, as it depends exclusively on third-parties.
```

Comment:
solvers usually participate in an auction like OFA so there's no need to mention the double submission problem. can remove this part